subsys/bluetooth/host/att.c

```diff
@@ -542,18 +543,19 @@ struct net_buf *bt_att_chan_create_pdu(struct bt_att_chan *chan, uint8_t op,
 	case ATT_RESPONSE:
 	case ATT_CONFIRMATION:
 		/* Use a timeout only when responding/confirming */
-		buf = bt_l2cap_create_pdu_timeout(NULL, 0, BT_ATT_TIMEOUT);
+		timeout = BT_ATT_TIMEOUT;
 		break;
 	default:
-		buf = bt_l2cap_create_pdu(NULL, 0);
+		timeout = K_FOREVER;
```

Comment:
is there a reason to not use `BT_ATT_TIMEOUT` for all allocations ?

Reply:
The ATT timeout is just from a request is sent to the response is received. It is ok to wait before sending a request (and the API is documented as blocking).